### DISCUSSION TITLE
[Bug] Can't split endpoint id into 2 parts  in Virtual Database or Virtual Cache query trace detail

### DISCUSSION THREAD

**Discussion by shilin66** [apache/skywalking#10818] [Q&A] [answered]
### Search before asking

- [x] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

**In version 9.4.0** .
 When i query Virtual Database Slow Statements trace detail , I got this error :  "Exception while fetching data (/endpoint) : Can't split endpoint id into 2 parts, b883efe43a8e45baae72dfd3b040c2bf.80.16843019257611193"。The similar error happened in query  virtual cache Slow Read Command or Slow Write Command。

### What you expected to happen

the endpointid could cause the error

### How to reproduce

1. I click this "查看相关trace"
![image](https://github.com/apache/skywalking/assets/39396378/d61c549a-3561-43f4-83f2-8b055ded2f40)
2. then i got this error
![image](https://github.com/apache/skywalking/assets/39396378/00b06e40-5a5a-425e-9f24-cb6dcb87bcc2)
The similar error happened in virtual cache

I readed source code about the query api , i found the method 
![image](https://github.com/apache/skywalking/assets/39396378/2ce36b29-f1ac-4618-ab01-b1023c96c373)
In the method analysisId , it will split the endponitId by "\_"。In fact , the endponitId  has no "\_"!

Is this a bug?

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
I don't know which cases, but this mean your endpoint name somehow is empty. It should be _blank by default.

**Reply by 12d** [reply]
I got the same error..how to handle it

**Comment by JorringHsiao** [comment]
The official [demo](http://demo.skywalking.apache.org/database) also had the same problem. 

![image](https://github.com/apache/skywalking/assets/39626963/5a29194d-772c-4fbb-a40f-8d4a31c10db5)

And the value of endpoint id is completely different, some cases are traceId and some cases are SQL statements.

**Comment by wu-sheng** [comment]
We are going to check this, we found why this happens. 9.5.0 release is cancelling.

**Reply by 1766095699** [reply]
The virtual cache's slow query also has the same issue. Have they fixed it together? (I haven't seen the front-end code, so I asked hhh)

**Comment by wu-sheng** [comment, marked as answer]
Fixed by https://github.com/apache/skywalking-booster-ui/pull/285
